### PR TITLE
Feature/view list pagination and searching

### PR DESCRIPTION
**Pull request by abdiel8real**
We can now watch the registers in the article table on the articles side-bar

### CODE REVIEW

File: app/Http/Controllers/ArticuloController.php [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+        $articulo -> precio_venta = $request -> precio_venta;
+        $articulo -> stock = $request -> stock;
+        $articulo -> descripcion = $request -> descripcion;
+        $articulo -> condicion = "1";
+        $articulo -> save();
+    }
+
+    public function update(Request $request)
+    {
+        if (!$request->ajax()) return redirect('/');
```
Comment (cesarureno) [inline app/Http/Controllers/ArticuloController.php:55]: Please use fill function to store article

File: app/Http/Controllers/ArticuloController.php [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+                'last_page'    => $articulos-> lastPage(),
+                'from'         => $articulos-> firstItem(),
+                'to'           => $articulos-> lastItem(),
+            ],
+            'articulos' =>$articulos
+        ];
+
+    }
+
+    public function store(Request $request)
```
Comment (cesarureno) [inline app/Http/Controllers/ArticuloController.php]: Please use fill function to store article

File: routes/web.php [hunk 3]
```diff
@@ -20,3 +20,9 @@
 Route::put("/categoria/actualizar", "CategoriaController@update");
```
Comment (cesarureno) [inline routes/web.php]: Remove initial '/' in all routes

File: app/Http/Controllers/ArticuloController.php (hunk 1 above)
Comment (abdiel8real) [reply app/Http/Controllers/ArticuloController.php:55]: Done

File: app/Http/Controllers/ArticuloController.php (hunk 2 above)
Comment (abdiel8real) [reply app/Http/Controllers/ArticuloController.php]: Done

File: routes/web.php (hunk 3 above)
Comment (abdiel8real) [reply routes/web.php]: Done and done

**Review by cesarureno — APPROVED** [review]

**Pull request merged**